Mathlib/Algebra/PUnitInstances.lean

```diff
@@ -97,29 +105,32 @@ theorem norm_unit_eq : normUnit x = 1 :=
   rfl
 #align punit.norm_unit_eq PUnit.norm_unit_eq
 
-instance : CanonicallyOrderedAddMonoid PUnit := by
+instance canonicallyOrderedAddMonoid: CanonicallyOrderedAddMonoid PUnit := by
   refine'
-        { PUnit.commRing, PUnit.completeBooleanAlgebra with
-          exists_add_of_le := fun _ _ _ => ⟨star, Subsingleton.elim _ _⟩.. } <;>
+        { PUnit.commRing,  PUnit.completeBooleanAlgebra with
+          exists_add_of_le := fun {_ _} _ => ⟨PUnit.unit, Subsingleton.elim _ _⟩.. } <;>
       intros <;>
     trivial
 
-instance : LinearOrderedCancelAddCommMonoid PUnit :=
-  { PUnit.canonicallyOrderedAddMonoid, PUnit.linearOrder with
-    le_of_add_le_add_left := fun _ _ _ _ => trivial }
+instance linearOrderedCancelAddCommMonoid: LinearOrderedCancelAddCommMonoid PUnit := by
+  refine'
+        { PUnit.canonicallyOrderedAddMonoid, PUnit.linearOrder with
+    le_of_add_le_add_left := fun _ _ _ _ => trivial
+    add_le_add_left := by intros; rfl }
 
 instance : LinearOrderedAddCommMonoidWithTop PUnit :=
-  { PUnit.completeBooleanAlgebra, PUnit.linearOrderedCancelAddCommMonoid with
+  { PUnit.completeBooleanAlgebra , PUnit.linearOrderedCancelAddCommMonoid with
     top_add' := fun _ => rfl }
 
 @[to_additive]
-instance : HasSmul R PUnit :=
+instance smul : SMul R PUnit :=
   ⟨fun _ _ => unit⟩
 
-@[simp, to_additive]
+@[to_additive (attr := simp, nolint simpNF)]
```

Comment:
You've added a `nolint simpNF` here -- accidental?

Reply:
yep -- copy n paste error